runners/core-java/src/test/java/org/apache/beam/runners/core/SimpleDoFnRunnerTest.java

```diff
@@ -391,6 +391,7 @@ public void testInfiniteSkew() {
 
     static final String TIMER_ID = "throwingTimerId";
 
+    @SuppressWarnings("unused")
```

Comment:
I talked to Kenn about this offline. It looks like the internal Google guava/errorpone has a feature that would allow us to avoid this. I opened an issue with error-prone here: https://github.com/google/error-prone/issues/2668

Reply:
Depending on how that goes we may want to hold off on actually turning this check on so we don't need to add >100 suppressions on `TimerId` and `StateId` checks. We could still move forward with removing unused variables that he check identifies either way.

Reply:
Ok I was able to make a change to error-prone to make it suppress this warning for TimerId and StateId: https://github.com/google/error-prone/commit/fad4116129fcbaa5e3418ff419f164552f75481a

We can't take advantage of that until it is in an error-prone release though. We _can_ go ahead and merge PRs that remove unused variables though, so that we're ready to turn the check on when that fix is released.

Reply:
I've removed all suppress inline annotations on stateId and timerId, also I've added a suppress annotation by class along with a TODO comment indicating to remove the line once a new version of error-prone is released.